packages/lodestar-validator/src/slashingProtection/utils.ts

```diff
@@ -0,0 +1,15 @@
+import {Epoch, Root} from "@chainsafe/lodestar-types";
+import {toHexString} from "@chainsafe/ssz";
+import {ZERO_HASH} from "@chainsafe/lodestar-beacon-state-transition";
+
+export function isEqualRoot(root1: Root, root2: Root): boolean {
+  return toHexString(root1) === toHexString(root2);
```

Comment:
Think it's better/faster to use config.types.Root.equals as it doesn't convert to string first and then compare strings.

Reply:
Good point! Could you link the implementation of `equals` for Root type?

Reply:
not 100% but https://github.com/ChainSafe/ssz/blob/master/src/backings/byteArray/abstract.ts#L20

Reply:
Suggestion applied in https://github.com/ChainSafe/lodestar/pull/1674/commits/1c10190d60378a812ea4a3be81873e054c46493c